--- ocaml-libvirt-0.6.1.2/examples/Makefile.in.orig	2014-08-08 15:32:57.792838736 +0100
+++ ocaml-libvirt-0.6.1.2/examples/Makefile.in	2014-08-08 15:33:33.556833763 +0100
@@ -27,8 +27,8 @@
 export LIBRARY_PATH=../libvirt
 export LD_LIBRARY_PATH=../libvirt
 
-BYTE_TARGETS	:= list_domains node_info get_cpu_stats
-OPT_TARGETS	:= $(BYTE_TARGETS:%=%.opt)
+#BYTE_TARGETS	:= list_domains node_info get_cpu_stats
+OPT_TARGETS	:= list_domains.opt node_info.opt get_cpu_stats.opt
 
 all: $(BYTE_TARGETS)
 
